### DISCUSSION TITLE
USDOT NAD Import?

### DISCUSSION THREAD

**Discussion by dammad** [osm-search/Nominatim#2777] [Geocoding around the world]
Is there a way to important the USDOT NAD database (https://www.transportation.gov/gis/national-address-database) to improve street addresses and numbers over what the TIGER DB provides?

**Comment by lonvia** [comment]
There is currently no support for adding external address data except for Tiger. In principle it is possible to generalize the existing code so that other address databases are supported. We even have an open issue for [supporting OpenAddresses](https://github.com/osm-search/Nominatim/issues/246). In practise, I consider this a feature which is more of interest to commercial applications than to the OSM community, which means it happens only if it is [funded](https://nominatim.org/funding.html).
